src/PactNet.Native/PactNet.Native.csproj

```diff
@@ -107,4 +107,10 @@
     <ProjectReference Include="..\PactNet\PactNet.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Http">
```

Comment:
Is this NuGet package still maintained? It looks very old and hasn't been updated for 2.5 years.

Reply:
Not sure about it